cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -100,6 +102,10 @@ func (s *server) Start(ctx context.Context, host component.Host) error {
 	if err := s.addArchiveStorage(&opts, &v2opts, host); err != nil {
 		return err
 	}
+
+	opts.Adjuster = adjuster.Sequence(querysvc.StandardAdjusters(s.config.MaxClockSkewAdjust)...)
+	v2opts.Adjuster = v2adjuster.Sequence(v2adjuster.StandardAdjusters(s.config.MaxClockSkewAdjust)...)
```

Comment:
@mahadzaryab1 did we miss adjusters altogether in query extension?

Reply:
@yurishkuro We didn;t miss the adjusters - they're applied by default in the query service (https://github.com/jaegertracing/jaeger/blob/main/cmd/query/app/querysvc/v2/querysvc/service.go#L82-L85). However, it looks like we weren't propagating the MaxClockSkewAdjust value all the way down to the query service.

Reply:
then it needs a bit of a bigger change to propagate the value via QuerySvcOptions -- @dnaka91 fyi